e2e-tests/commands/shared-commands/assertions/check-text-and-value.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * checkTextAndValue
+ * Check the text and value of a selector
+ * @param {Function} textSelector: Cypress selector for text
+ * @param {String} expectedText: Expected text
+ * @param {valueSelector} selector: Cypress selector for value
+ * @param {String} expectedValue: Expected value
+ */
+const checkTextAndValue = (textSelector, expectedText, valueSelector, expectedValue) => {
```

Comment:
Can we have the params structured as an object?

Also line 6 has incorrect param type